pages/data-cleanup.tsx

```diff
@@ -670,21 +670,16 @@ export default function DataCleanupPage(props) {
                         dropDownItems={dropdownData.genderTypes}
                         handleChange={handleChange}
                       ></DropDownField>
-
-                      <DateField
-                        label="Date of Birth"
-                        fieldName="dob"
-                        fieldValue={formData.contactRecord?.dob}
-                        handleDateChange={handleDateChange}
-                      ></DateField>
                     </>
                   ) : (
-                    <TextField
-                      label="Company Name"
-                      fieldName="company_name"
-                      fieldValue={formData.contactRecord?.company_name}
-                      handleChange={handleChange}
-                    ></TextField>
+                    <>
+                      <TextField
+                        label="Company Name"
+                        fieldName="company_name"
+                        fieldValue={formData.contactRecord?.company_name}
+                        handleChange={handleChange}
+                      ></TextField>
+                    </>
```

Comment:
The blank element tags are only needed if you're returning more than one element. A JSX return, even from a ternary, always needs to be wrapped in a single element. Since there's only one element here it's not needed